### DISCUSSION TITLE
Will it be open source?

### DISCUSSION THREAD

**Discussion by paulorwd** [easypanel-io/community#72] [General]
I was looking for a project along these lines to implement in my company, I would like to carry out several implementations and I believe it could benefit the community (SSO, Others git, others image repositories, project permissions, Kubernetes, auto webhook configuration, etc...)
Open source is a possibility?

**Comment by deiucanta** [comment]
Unfortunately, there are no plans to open source the core functionality. We have open-sourced the [templates](https://github.com/easypanel-io/templates) and the [builders](https://github.com/easypanel-io/builders) (not very useful at the moment). However, most of the suggested features are on the roadmap.

**Reply by paulorwd** [reply]
I understand, is there prediction?  I can't access the roadmap

**Reply by deiucanta** [reply]
I moved all the ideas and the roadmap to https://feedback.easypanel.io

**Reply by jamesbruice** [reply]
@deiucanta I cant access that link

**Reply by deiucanta** [reply]
@jamesbruice fixed now
